src/Pim/Bundle/CatalogBundle/tests/integration/ElasticSearch/IndexConfiguration/AbstractPimCatalogIntegration.php

```diff
@@ -49,11 +58,16 @@ protected function setUp()
      */
     private function resetIndex()
     {
-        if ($this->ESClient->indices()->exists(['index' => self::INDEX_NAME])) {
-            $this->ESClient->indices()->delete(['index' => self::INDEX_NAME]);
+        $conf = $this->esConfigurationLoader->load();
+
+        try {
+            $this->esClient->deleteIndex();
```

Comment:
You could take advantage of the hasIndex function of the library (adding a "passe-plat" it into our class)
```
if ($this->esClient->hasIndex()) {
  $this->esClient->deleteIndex()
}
```